app/states/playerRefactor.js

```diff
@@ -0,0 +1,155 @@
+import Prefab from './entityPrefab'
+
+// client side class for Playable Characters
+export default class Player extends Prefab {
+  constructor(x, y, key) {
+    // call it's superclass constructor
+    super(x, y, key)
```

Comment:
Change constructor and super arguments:
constructor(game, name, position, spriteKey)
super(game, name, position, spriteKey)